build.gradle

```diff
@@ -0,0 +1,126 @@
+apply plugin: 'java'
+// For buildscript -> repositories -> maven
+apply plugin: 'maven'
+apply plugin: 'application'
+apply plugin: "org.sonarqube"
+// Add automated release process
+apply plugin: 'net.researchgate.release'
+
+sourceCompatibility = 1.7
+targetCompatibility = 1.7
+
+// Load the version number. This file sits in resources to include it into the jar. Because the version
+// is needed at runtime.
+
+// Configure the maven repository deployment
+install {
+    repositories.mavenInstaller {
+        // Set the version
+        pom.version = version
+        // Set the group/namespace for the maven repository deployment.
+        pom.groupId = 'jsr223'
```

Comment:
Yes. It will be like this: jsr223:jsr223-perl:0.1.0
I just checked, when I upload to our nexus repository.